### PR TITLE
Add docs for top level job inputs

### PR DESCRIPTION
**Pull request by dpeng817**
Added as an advanced example since I think the use case is potentially niche.

### CODE REVIEW

**Review by sryza — COMMENTED** [review]
Great to have this documented. This page doesn't introduce `execute_in_process`, so it might be confusing to introduce it here? Maybe makes more sense on the Job Execution page? Or on the testing page?

File: docs/content/concepts/ops-jobs-graphs/jobs-graphs.mdx [hunk 1]
```diff
@@ -398,6 +398,40 @@ def alias():
...
+def op_with_input(x):
+    return do_something(x)
+
+
+@graph
+def wires_input(x):
+    op_with_input(x)
+```
+
+I can turn the graph into a job by calling <PyObject object="GraphDefinition" method="to_job" />, and can provide a value to the input `x` using the `input_values` argument.
```
Comment (erinkcochran87) [inline docs/content/concepts/ops-jobs-graphs/jobs-graphs.mdx:419]:
```suggestion
Turn the graph into a job by calling <PyObject object="GraphDefinition" method="to_job" /> and providing a value to the input `x` using the `input_values` argument:
```

Second person is okay in the docs, but let's refrain from first. I changed this to use an understood subject (`you`) and make it second person.

File: docs/content/concepts/ops-jobs-graphs/jobs-graphs.mdx [hunk 2]
```diff
@@ -398,6 +398,40 @@ def alias():
...
+
+```python file=/concepts/ops_jobs_graphs/jobs.py startafter=start_top_level_input_job endbefore=end_top_level_input_job
+the_job = wires_input.to_job(input_values={"x": 5})
+```
+
+I can also provide input values using <PyObject object="JobDefinition" method="execute_in_process" /> or <PyObject object="GraphDefinition" method="execute_in_process" />.
```
Comment (erinkcochran87) [inline docs/content/concepts/ops-jobs-graphs/jobs-graphs.mdx:425]:
```suggestion
You can also provide input values using <PyObject object="JobDefinition" method="execute_in_process" /> or <PyObject object="GraphDefinition" method="execute_in_process" />:
```

Same thing here, re: first vs second person.

**Comment by dpeng817** [comment]
> Great to have this documented. This page doesn't introduce `execute_in_process`, so it might be confusing to introduce it here? Maybe makes more sense on the Job Execution page? Or on the testing page?

Will link the docs to execute_in_process, but I think this should probably

**Review by erinkcochran87 — APPROVED** [review]
:shipit: Looks good!

**Pull request merged**